src/App/Document.h

```diff
@@ -563,6 +564,9 @@ class AppExport Document : public App::PropertyContainer
 
     std::string oldLabel;
     std::string myName;
+
+    /// Time document was last modified
+    std::time_t myEditTime;

```

Comment:
To me "edit time" sounds like "last time I made a change to my open document", but what you are really after is "lastSavedTime", right?

Reply:
The name can be workshopped. I think the key is the "my" i.e. it is the time used as reference by the `Document` object.